mantidproject/quickBayes
# Delete Fortran :D

This PR is removing the old code (Fortran and the direct translation). It also cleans up the code base, removing unused files and renames/restructures the code to suit the new code.

The "help" method of Python also works with the new package
```
>>> help("quickBayes")
Help on package quickBayes:

NAME
    quickBayes

PACKAGE CONTENTS
    fit_engines (package)
    fit_functions (package)
    log_likelihood
    utils (package)
    workflows (package)
```

To test:

see that the release pipieline is working: https://pypi.org/project/quickBayes/1.0.0b3/#history (windows and linux for 1.0.0b3) from this commit https://github.com/mantidproject/quickBayes/pull/57/commits/06e58808b708a8a628ba153f06ac981d48cf682b

Review the code and see that the Fortran is gone
